dns.tf

```diff
@@ -69,7 +69,8 @@ variable "subdomain" {
     "phage.usegalaxy.eu",
     "aqua.usegalaxy.eu",
     "earth-system.usegalaxy.eu",
-    "eirene.usegalaxy.eu"
+    "eirene.usegalaxy.eu",
+    "microbiology.usegalaxy.eu"
```

Comment:
Did you check the `count` matches under `resource "aws_route53_record" "subdomains"`?

Reply:
Good catch! Thanks! Updated it now.